### DISCUSSION TITLE
Disable default user interactions completely

### DISCUSSION THREAD

**Discussion by omar-shahid** [xyflow/xyflow#2417] [Q&A] [answered]
Hi React Flow Community,
Thank you so much to the creator and contributors of this library for putting this out! Which otherwise would be a nightmare to create from scratch 😄 

I have got into some issues while disabling user interactions. I'm using react flow with more customized code, so the user cannot use any of the default interactions. I have found and added classes like "nodrag nowheel" in custom nodes, but there are still some things that don't get removed:

1. Cursor always is in the "hand" state, giving a signal that nodes can be moved (even though it is disabled) and the user can't select anything in nodes.
2. Even though I have set "nodesDraggable={false}" in Main React Flow Component, users can move nodes when multiple nodes are selected.
3. If user hovers over the handle, the cursor transforms to plus icon, and user can drag a new edge from it.
![image](https://user-images.githubusercontent.com/40920737/189289618-1963027f-5e67-4893-b377-88f4acf1a444.png)
![image](https://user-images.githubusercontent.com/40920737/189289543-5beeaae8-2265-4289-86bf-02b55b9cb45e.png)

In short, how can I disable all of the default user interactions which are shipped with react flow? I'm using it for visualizing purpose only and all of the transformations of nodes and edges are done from code. 
Thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
Have you checked the [`Interaction`](https://reactflow.dev/docs/examples/interaction/interaction-props/) example?
All props related to interaction are demonstrated there.

For example - if you want to disable "multi-select", turn off `elementsSelectable`.

**Reply by omar-shahid** [reply]
Thanks! This answers almost all questions, except "Selecting contents inside node". How can I make them behave like normal piece of content with all cursor interactions?

**Reply by bcakmakoglu** [reply]
That's an issue that someone else stumbled upon as well.
The only solution for now, at least that came to my mind is, wrap your custom node in a div and apply `pointer-events: all` 😅
Disabling the interactions also disables all pointer events on a node.

**Reply by omar-shahid** [reply]
Thank you so much for your assistance!
